wfl/configset.py

```diff
@@ -104,6 +104,9 @@ def __init__(self, abcd_conn=None,
         elif input_files is not None:
             # fix up files
             self.input_files = []
+            if isinstance(input_files, Path):
+                # Path.glob() below takes string as argument
+                input_files = str(input_files)
```

Comment:
If we convert `file_root` from str **to** Path before storing it, why are we doing the opposite here?  Is there a reason not to store all files/paths as Path objects, and converting to str when necessary?  We will need to do it when passing to `glob.glob()`, but `ase.io.read` can already accept Paths.

Reply:
Yes, this is redundant. Forgot to remove it after changing to `glob.glob()` below.